### DISCUSSION TITLE
Can't find Archer SR10+, SR8+

### DISCUSSION THREAD

**Discussion by Boebe62** [FrSkyRC/ETHOS-Feedback-Community#4899] [Q&A] [answered]
I updated to 16.0 and now I want to update my stabilized receivers. I can't find them on the download page in Ethos Suite.
Any suggestions ?

<img width="501" alt="Scherm­afbeelding 2024-12-22 om 11 57 21" src="https://github.com/user-attachments/assets/ddbd73c4-84bf-4e4d-be4b-7822950afb1d" />

**Comment by mgsanna** [comment, marked as answer]
Hi You have to go to AP series (archer plus) to find them.....
<img width="272" alt="Screenshot 2024-12-22 134722" src="https://github.com/user-attachments/assets/b2ccafe2-e473-4b0d-93a8-aeec678dd4bd" />
